tests/lava/lib/optimization/solvers/lca/test_fixed.py

```diff
@@ -0,0 +1,248 @@
+#  Copyright (C) 2023 Battelle Memorial Institute
+#  SPDX-License-Identifier: BSD-2-Clause
+#  See: https://spdx.org/licenses/
+
+import unittest
+
+import numpy as np
+from lava.magma.core.run_configs import Loihi1SimCfg
+from lava.magma.core.run_conditions import RunSteps
+from lava.proc.io import sink
+
+from lava.lib.optimization.solvers.lca.process import LCA1Layer, LCA2Layer
+from lava.lib.optimization.solvers.lca.lca_neuron.process import LCANeuron
+from lava.lib.optimization.solvers.lca.accumulator.process import \
+    AccumulatorNeuron
+from lava.lib.optimization.solvers.lca.models import LCA1LayerModelFixed
+from lava.lib.optimization.solvers.lca.accumulator.models import \
+    PyAccumulatorFixed
+from lava.lib.optimization.solvers.lca.lca_neuron.models import \
+    PyLCANeuronFixed
+
+
+class TestLCAFixed(unittest.TestCase):
```

Comment:
Add a brief docstring for unit-tests. This helps in using the unit-tests also as a teaching/learning tool.